server/sonar-web/src/main/js/apps/issues/conciseIssuesList/ConciseIssuesListHeader.js

```diff
@@ -29,18 +29,23 @@ type Props = {|
   loading: boolean,
   onBackClick: () => void,
   onReload: () => void,
+  displayBackButton?: boolean,
   paging?: Paging,
   selectedIndex: ?number
 |};
 */
 
 export default function ConciseIssuesListHeader(props /*: Props */) {
   const { paging, selectedIndex } = props;
+  const shouldDisplayBackButton =
+    typeof props.displayBackButton === 'undefined' || props.displayBackButton;
```

Comment:
I'd instead do:
```javascript
const { displayBackButton = true, paging, selectedIndex } = props;
```

or change the property to `hideBackButton` and then it's fine to check it:
```javascript
{!props.hideBackButton && (
  <BackButton className="pull-left" onClick={props.onBackClick} />
)}
```